guides/_subsections/using-smart-triggers.md

```diff
@@ -0,0 +1,92 @@
+      </pre>
+      <p>
+      The preceding example trigger instructs the agent to start a JFR recording if the ProcessCpuLoad metric has a value greater than 0.2 for a duration of more than 30 seconds: This example also instructs the agent to use the profile event template for the JFR recording.
+      </p>
+    </details>
+  </li>
+  <li>
+    <details>
+      <summary>Using a JVM system property flag</summary>
+      <p>
+The following example shows how to specify a simple custom trigger by using a JVM system property flag:
+      </p>
+      <pre>
+  -Dcryostat.agent.smart-trigger.definitions="[ProcessCpuLoad > 0.2 ; TargetDuration > duration(\"30s\")]~profile"
+      </pre>
+      <p>
+      This example uses the same custom trigger criteria as the preceding example.
+      </p>
+    </details>
+  </li>
+  <li>
+    <details>
+      <summary>Using an environment variable</summary>
+      <p>
+The following example shows how to specify a simple custom trigger by using an environment variable:
+      </p>
+      <pre>
+  - name: CRYOSTAT_AGENT_SMART_TRIGGER_DEFINITIONS
+    value: "[ProcessCpuLoad > 0.2 ; TargetDuration > duration(\"30s\")]~profile"
+      </pre>
+      <p>
+This example uses the same custom trigger criteria as the preceding examples.
+      </p>
+    </details>
+  </li>
+</ol>
+
+#### [General Syntax Rules for Custom Triggers](#general-syntax-rules-for-custom-triggers)
+Consider the following syntax guidelines for defining custom triggers:
+<ol>
+    <li>A custom trigger definition must consist of both an expression that defines the overall trigger condition and the name of an event template that is used for the JFR recording.</li>
+    <li>The entire trigger expression must be enclosed in square brackets. For example: <pre>[ProcessCpuLoad > 0.2 ; TargetDuration < duration("30s")]</pre></li>
+    <li>The name of the event template must be defined after the trigger expression and preceded by a tilde <code>(~)</code> character. For example: <pre>~profile</pre></li>
+    <li>A trigger expression can consist of one or more constraints and a target duration. The set of constraints and target duration must be separated by a semicolon <code>(;)</code> character.</li>
+    <li>Each constraint must include: the name of an MBean counter; a relational operator such as <code> ></code> (greater than), <code>=</code> (equal to), <code> < </code>(less than), and so on; and a specified value. The type of relational operator and value that you can specify depends on the associated MBean counter type. For example: <pre>ProcessCpuLoad > 0.2</pre></li>
+    <li>Constraints can be grouped together by using logical operators such as <code>&&</code> (AND), <code>|| </code> (OR), or <code> ! </code> (NOT) logic. For readability and clarity around the order of operations and operator precedence, grouped constraints may be enclosed in round brackets, but this is not a requirement. For example:
```

Comment:
Here there are some spaces captured inside the code blocks, like `<code>|| </code>` - I think this should just be `<code>||</code>`?